org.eclipse.dartboard/src/org/eclipse/dartboard/util/StringUtil.java

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************

```

Comment:
I've added Google Guava in https://github.com/eclipse/dartboard/commit/81afb33efd409266bcd1f6b44ded83a6494fcde8. Could you use the String utils from them instead of adding our own?